docs/video-processing/functionalities/set-coordinates-import-options.md

```diff
@@ -22,16 +28,56 @@ nav_order: 3
 
 ## Reference coordinates
 
+Modify these settings according to the structure of the reference coordinates file to import.
+
 <a href="../../assets/images/gui/VP-panel-molcoord-transf-impopt-refcoord.png"><img src="../../assets/images/gui/VP-panel-molcoord-transf-impopt-refcoord.png" style="max-width: 274px;"/></a>
 
+Reference coordinates are coordinates localized in each channel.
+Channel-specific coordinates can be written column-wise or row-wise in file.
+To read coordinates column-wise, select **(a)**. 
+To read coordinates row-wise by selecting **(e)**.
+
+### Column-wise
+{: .no_toc }
+
+Coordinates (x,y) of one single molecule in individual channels are written on the same line and in different columns.
+
+The number of header lines in the file must be set in **(b)**.
+
+For each channel, give in **(c)** and **(d)** the column indexes in the file where channel-specific x- and y- coordinates are written.
+
+
+### Row-wise
+{: .no_toc }
+
+Coordinates (x,y) of one single molecule in individual channels are written on different line and in same columns.
+
+This is the default import format and is based on the structure of 
+[.map files](../../output-files/map-reference-coordinates.html).
```

Comment:
@mca-sh dead link: should this be `map-mapped-coordinates.html`?

Reply:
Oh yes, thank you. I think I did the mistake many times...